articles/azure/asz-sco

```diff
@@ -0,0 +1,262 @@
+A storage account can contain any number of tables, and a table can contain any number of entities, up to the capacity limit of the storage account.
+
+> [!NOTE]
+> All storage in Azure Stack is taken from a common pool. When designing your service for UKCloud for Microsoft Azure, consider the best usage of your storage. For example, if you have a large object storage requirement, consider using the UKCloud's Cloud storage platform as part of your solution. 
+ 
+## Service Management
+- API 
+
+- UKCloud for Microsoft Azure Marketplace – provides access to pre-created images which help speed up application delivery 
+
+- Azure Resource Manager – allows customers to write templates to deploy, update and delete UKCloud for Microsoft Azure resources in a single coordinated operation 
+
+- VM Extensions – UKCloud for Microsoft Azure supports virtual machine extensions that provide post deployment management for automation (for example, Docker) 
+ 
+## Network
+
+UKCloud manage the physical firewalls that face public and secure networks.
+
+Customers can deploy mutliple gateway devices and configure the following using either the Azure Stack Portal GUI or API:
+
+- Virtual networks – UKCloud for Microsoft Azure allows customers to configure virtual private clouds, and allows them to create and manage their private address space within Azure. This can be configured via the UI, API and other tooling 
+- Load balancers (IPv4 only) – customers can deploy basic load balancers o VPN gateway – the UKCloud for Microsoft Azure VPN gateway can be used to create site-to-site VPNs to remote infrastructure 
+- Routing – basic custom defined routing can be configured on virtual networks 
+- SSL VPN?
+
+**Internet-facing solutions** have 8 usable public IP addresses. You can ask for additional public IP addresses via a Service Request. We reserve the right to decline the request if you have spare capacity in your existing deployment.
+
+## Advanced features 
+- Key Vault. UKCloud for Microsoft Azure provides cryptographic secret storage for applications and services. These services can be consumed by both portal and API.  
+https://docs.microsoft.com/en-us/azure/azure-stack/user/azure-stack-kv-intro
+
+## Protection options
+
+Customers are responsible for designing their own disaster recovery and backup solutions as part of their adoption of UKCloud for Microsoft Azure.
+
+## Service availability
+
+The availability of your virtual machines in UKCloud for Microsoft Azure follows the same principles of the Microsoft Azure Public Cloud, where customers achieve a higher service level availability based on how they architect the solution. 
+
+The service levels are as follows:
+
+- 99.90% (a single VM deployed as a single instance deployed inside and availability zone)
+- 99.95% (a single VM with two instances deployed inside an availability zone)
+
```

Comment:
Shouldn't we be using zone rather than availability zone (in which case it should be a not an)